contracts/lib/AddressArrayUtils.sol

```diff
@@ -0,0 +1,346 @@
+pragma solidity 0.4.25;
```

Comment:
Can we add a link to the cryptofin repo? Also could we pull in their test files as well (we may put it in a new test folder)

Reply:
+1

Reply:
what do you mean a link to the cryptofin repo?